apps/to-do-api/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "to-do-api",
+  "version": "0.0.0",
+  "description": "Azure Function REST API for To Do List",
+  "license": "MIT",
+  "main": "./dist/main.js",
+  "scripts": {
+    "clean": "shx rm -rf ./dist ./src/generated",
+    "build": "npm run generate && tsc",
+    "typecheck": "npm run build --noemit",
+    "lint": "eslint --fix",
+    "lint:check": "eslint",
+    "lint:api": "sh api-lint.sh",
+    "generate": "shx mkdir -p ./src/generated/definitions/internal && gen-api-models --api-spec ./docs/openapi.yaml --no-strict --out-dir ./src/generated/definitions/internal",
```

Comment:
- why are you generating these files?
- `mkdir` is not needed, `gen-api-models` creates the directory automatically

Reply:
The files are generated in order to have a type, generated starting from the OpenAPI, to use in the function's handler.

`generate` command updated removing the `mkdir`.